scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -130,6 +130,15 @@ ruleTester.run('protractor-practices', rule, {
         message: 'Please do not use .then(), consider async/await instead',
       }],
     },
+    {
+      code:
+      `genderList.filter(function() {
+        gender = 'female';
+      })`,
```

Comment:
Can you make these test cases more comprehensive? For example:

* A comment that includes "filter" should pass
* Reading the `filter` attribute of an object (but not calling it as a function) should pass
* Passing a function to `filter()` that was defined elsewhere should fail (e.g. `var func = function() {...} ... elems.filter(myFunc)`